algorithm/sorting-algo-python/quick_sort.py

```diff
@@ -8,7 +8,7 @@ def quick_sort(arr):
     return quick_sort(left) + middle + quick_sort(right)
 
 # Driver code to test above
-arr = [3, 6, 8, 10, 1, 2, 1]
+arr = [10, 2, 7, 3, 5, 8, 1, 5, 74, 2, 2, 7, 2, 8, 52, 6, 74, 2]
 print("Original array:", arr)
 sorted_arr = quick_sort(arr)
-print("Sorted array:", sorted_arr)
+print("Sorted array:", sorted_arr)
```

Comment:
Add EOL. Formatting with Prettier will do it automatically.

Reply:
Prettier was not available for python, so I used black